sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.students.controller.getCompletedApplicationDetails.e2e-spec.ts

```diff
@@ -313,9 +319,169 @@ describe("ApplicationStudentsController(e2e)-getCompletedApplicationDetails", ()
             secondDisbursement.disbursementScheduleStatus,
         },
         assessmentTriggerType: AssessmentTriggerType.RelatedApplicationChanged,
+        hasFeedbackError: false,
       });
   });
 
+  it(
+    "Should get application details with feedback error status as false when the application has one or more feedback errors but none of them" +
+      " block funding and the offering intensity is part-time.",
+    async () => {
+      // Arrange
+      const application = await saveFakeApplicationDisbursements(
+        appDataSource,
+        { student },
+        {
+          applicationStatus: ApplicationStatus.Completed,
+          offeringIntensity: OfferingIntensity.partTime,
+          firstDisbursementInitialValues: {
+            coeStatus: COEStatus.completed,
+            disbursementScheduleStatus: DisbursementScheduleStatus.Sent,
+          },
+        },
+      );
+      const [firstDisbursement] =
+        application.currentAssessment.disbursementSchedules;
+      // Feedback error which does not block funding.
+      const eCertFeedbackError = await db.eCertFeedbackError.findOne({
+        select: { id: true },
+        where: {
+          blockFunding: false,
+          offeringIntensity: OfferingIntensity.partTime,
+        },
+      });
+      // Create fake disbursement feedback error.
+      const feedbackError = createFakeDisbursementFeedbackError({
+        disbursementSchedule: firstDisbursement,
+        eCertFeedbackError,
+      });
+      await db.disbursementFeedbackErrors.save(feedbackError);
+      const endpoint = `/students/application/${application.id}/completed`;
+      const token = await getStudentToken(
+        FakeStudentUsersTypes.FakeStudentUserType1,
+      );
+      // Act/Assert
+      await request(app.getHttpServer())
+        .get(endpoint)
+        .auth(token, BEARER_AUTH_TYPE)
+        .expect(HttpStatus.OK)
+        .expect({
+          firstDisbursement: {
+            coeStatus: COEStatus.completed,
+            disbursementScheduleStatus: DisbursementScheduleStatus.Sent,
+          },
+          assessmentTriggerType: application.currentAssessment.triggerType,
+          hasFeedbackError: false,
+        });
+    },
+  );
+
+  it(
+    "Should get application details with feedback error status as true when the application has one or more feedback errors" +
+      " that block funding and the offering intensity is part-time.",
+    async () => {
+      // Arrange
+      const application = await saveFakeApplicationDisbursements(
+        appDataSource,
+        { student },
+        {
+          applicationStatus: ApplicationStatus.Completed,
+          offeringIntensity: OfferingIntensity.partTime,
+          firstDisbursementInitialValues: {
+            coeStatus: COEStatus.completed,
+            disbursementScheduleStatus: DisbursementScheduleStatus.Sent,
+          },
+        },
+      );
+      const [firstDisbursement] =
```

Comment:
As a suggestion, we can have at least one between all new E2Es checking for the second disbursement with the feedback errors.